src/API/FeedState.php

```diff
@@ -186,8 +193,15 @@ private function add_local_feed_state( $result ) {
 				$status       = 'success';
 				$status_label = esc_html__( 'Up to date', 'pinterest-for-woocommerce' );
 				$extra_info   = sprintf(
-					/* Translators: %1$s Time string, %2$s total number of products */
-					esc_html__( 'Successfully generated %1$s ago - Wrote %2$s products to %3$sfeed file%4$s', 'pinterest-for-woocommerce' ),
+					esc_html(
+						/* Translators: %1$s Time string, %2$s total number of products */
```

Comment:
While we're editing this, we should also include 3 and 4 explanations in the translators comment.

Also, the word "translators" in the comment is supposed to be lowercased.

```suggestion
						/* translators: 1: Time string, 2: total number of products, 3: opening anchor tag, 4: closing anchor tag */
```